tools/ci/docker/ydoc-server-polyglot/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM jeanblanchard/alpine-glibc
```

Comment:
why this particular one? There are more popular editions

Reply:
It didn't work with `frolvlad/alpine-glibc`

Reply:
Oh, that's a bit surprising.